aim/db/model_base.py

```diff
@@ -130,10 +130,26 @@ def to_attr(self, session):
         Child classes should override this method to specify a custom
         mapping of model properties to resource attributes.
         """
-        return {k: self.get_attr(session, k) for k in dir(self)
-                if (not k.startswith('_') and
+        # "object_dict" and "tree uses LargeBinary sqlalchemy datatype.
+        # LargeBinary sqlalchemy datatype needs "bytes-like" object
```

Comment:
nit: missing '.' at the end of the setence.

Reply:
'.' is missing because line hasn't ended, I will de-capitalize the next line "And"